lib/axiom/fs/stdio.js

```diff
@@ -47,3 +47,15 @@ export var Stdio = function(stdin, stdout, stderr, signal, ttyin, ttyout) {
 };
 
 export default Stdio;
+
+/**
+ * @return {void}
+ */
+Stdio.prototype.close = function() {
+  this.stdin.close();
```

Comment:
(Sorry I am reading this sequentially, so my comment here will probably be duplicated somewhere else):
ReadableStream should not have a "close" method. We are following the node.js stream design (see https://nodejs.org/api/stream.html). You can have a "close" or "end" event, but the *consumer* of a readable stream should not be able to close it.